Assets/MRTK/SDK/Experimental/PulseShader/Scripts/HideOnDevice.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HideOnDevice : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+#if !UNITY_EDITOR
```

Comment:
Can this use `Application.isEditor` instead?

Reply:
Updated to use if(!Application.isEditor)